tox.ini

```diff
@@ -63,9 +63,9 @@ commands =
     pylint celery_utils
     pylint tests
     pylint test_utils
-    pylint --py3k celery_utils
-    pylint --py3k tests
-    pylint --py3k test_utils
+    pylint celery_utils
```

Comment:
I think that we can remove these lines since they are repeating the previous lines